templates/superwerker.template.yaml

```diff
@@ -61,14 +61,14 @@ Metadata:
 Parameters:
   Domain:
     Type: String
-    Description: Domain used for root mail feature
+    Description: Domain used for root mail feature. Please see https://github.com/superwerker/superwerker/tree/main/docs/technical_faq.md for more information
```

Comment:
@Junkern could you update those links, then we're good to go.

Reply:
Updated the links